tests/components/unifi/test_controller.py

```diff
@@ -379,7 +379,21 @@ async def test_wireless_client_event_calls_update_wireless_devices(
     hass, aioclient_mock, mock_unifi_websocket
 ):
     """Call update_wireless_devices method when receiving wireless client event."""
-    await setup_unifi_integration(hass, aioclient_mock)
+    client_1_dict = {
```

Comment:
Mocking was missing.  It was blowing up but we never saw it because it happened in a call_soon

Reply:
Nice catch!